python/src/deltachat/eventlogger.py

```diff
@@ -1,28 +1,88 @@
+import deltachat
 import threading
-import re
 import time
+import re
 from queue import Queue, Empty
-from .hookspec import hookimpl
+from .hookspec import account_hookimpl, global_hookimpl
+
+
+@global_hookimpl
+def account_init(account):
+    # send all FFI events for this account to a plugin hook
+    def _ll_event(ctx, evt_name, data1, data2):
+        assert ctx == account._dc_context
+        ffi_event = FFIEvent(name=evt_name, data1=data1, data2=data2)
+        account._pm.hook.process_ffi_event(
+            account=account, ffi_event=ffi_event
```

Comment:
`account._pm` is a `PerAccount` instance, `.process_ffi_event()` expects only `ffi_event` arg

(I hope I am not misunderstanding all this pluggy stuff ;)

Reply:
fixing it.

Reply:
was this fixed? conversation is marked as solved but code remains the same ;)

Reply:
ah didn't explain: 
- introduced the per-account plugin manager so if you register there you don't need to disambigutate with other accounts in each hook implementation 
- i am still passing the account to allow cross-account process_ffi_event callbacks (i.e. on some more global manager object).  You can implement global and per-account hooks on the same plugin object.